query/src/main/scala/hmda/query/projections/institutions/InstitutionDBProjection.scala

```diff
@@ -37,6 +38,9 @@ class InstitutionDBProjection extends HmdaActor {
     case CreateSchema =>
       repository.createSchema().map(_ => InstitutionSchemaCreated()) pipeTo sender()
 
+    case DeleteSchema =>
```

Comment:
I thought this might be useful as part of this [ticket](https://github.com/cfpb/hmda-platform/issues/787). It might be that we don't need to expose this functionality at the actor level